app/templates/index.html

```diff
@@ -68,6 +68,7 @@ <h1><code>query-server</code></h1>
                             <button type="submit" value="quora" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/quora_icon.png') }}" width="30px" alt="Quora Icon"> Quora</button>
                             <button type="submit" value="parsijoo" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/parsijoo_icon.png') }}" width="30px" alt="Parsijoo Icon"> Parsijoo</button>
                             <button type="submit" value="mojeek" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/mojeek_icon.png') }}" width="30px" alt="Mojeek Icon"> Mojeek</button>
+                            <button type="submit" value="hqihooso" class="btn btn-lg  search btn-outline"><img src="{{ url_for('static', filename='images/qihooso_icon.png') }}" width="30px" alt="QihooSo Icon"> QihooSo</button>
```

Comment:
#327 must be fixed before we propagate more workarounds.

Reply:
No need of this

Reply:
I didn't get it @bhaveshAn

Reply:
Using `hqihooso` instead of `qihooso` has nothing to do with the working.
We can merge this PR now and when all the serach engine will use `full names` instead of `initial` then we can shift to `qihooso` .

Also, using `hqihooso` instead of `qihooso` beacause its using `q` keyword for `Quora`

Reply:
We should merge #327 first and then this PR, it's already delayed much.

Reply:
Please change "hqihooso" to "qihooso"